pkg/sentry/platform/ring0/kernel.go

```diff
@@ -58,6 +63,11 @@ func kernelSyscall(c *CPU) {
 
 // kernelException is a trampoline.
 //
+// When in amd64, it is called with %rip on the upper half, so it can
```

Comment:
Optional: you can just reference the above documentation, which can help keep them in sync. E.g.

// kernelException is subject to the same restrictions as kernelSyscall above.